task.go

```diff
@@ -19,6 +27,54 @@ type Task struct {
 	UpdatedAt         time.Time `json:"updated_at"`
 }
 
+// getAllTasks forcely returns all Harvest Tasks existed on the Account
+func (a *API) getAllTasks(args Arguments) (tasks []*Task, err error) {
+	tasks = make([]*Task, 0)
+
+	next := func() ([]*Task, tasksPaginated, error) {
+		return a.getPaginatedTasks(args)
+	}
+
+	var pageTasks []*Task
+	for ok := (next != nil); ok; ok = (next != nil) {
+		pageTasks, next, err = next()
+		if err != nil {
+			return nil, err
+		}
+
+		tasks = append(tasks, pageTasks...)
+	}
+
+	return tasks, err
+}
+
+// getPaginatedTasks returns Harvest Tasks as it returns Havest (divided by pages)
+func (a *API) getPaginatedTasks(args Arguments) (tasks []*Task, next tasksPaginated, err error) {
+	tasks = make([]*Task, 0)
+	pagedResponse := &tasksResponse{}
+	err = a.Get("/tasks", args, pagedResponse)
+
+	if pagedResponse.NextPage != 0 {
+		next = func() (tasks []*Task, next tasksPaginated, err error) {
+			args["page"] = strconv.Itoa(pagedResponse.NextPage)
+			return a.GetTasks(args)
+		}
+	}
+
+	return pagedResponse.Tasks, next, err
+}
+
+// GetTasks returns list Harvest Tasks.
+// * args["paginated"] = "true" - is used to get ALL tasks without breaking to pages
+func (a *API) GetTasks(args Arguments) (tasks []*Task, next tasksPaginated, err error) {
+	if args["paginated"] == "false" {
```

Comment:
Nitpick: as Harvest API doesn't have `paginated` parameter and yet `args` to be send to it as-is, so we could state function behavior change in a second param (e.g. some nullable options struct).

Reply:
See https://github.com/sergeykuzmich/harvestapp-sdk/pull/8/commits/c33815841402a9db51091a27d186435d89f95453 with suggested approach.